RELEASE_README.md

```diff
@@ -8,10 +8,15 @@ SigLens executable should support: `linux/arm64, linux/amd64, darwin/arm64, and
 
 ## SigLens Release Steps
 
-1. Create a pull request to develop which increment the verion number `SigLensVersion` located in `pkg/config/version.go` ***This step is critical. Failure to increment this number will lead to failure in creating github release***
-2. Merge develop to main using Create a Merge Commit
-   Do NOT Squash and Merge
-3. GitHub Actions will take care of the following builds:
+1. In the `develop` branch, increment the version number `SigLensVersion` located in `pkg/config/version.go`. The version number in `develop` will have a suffix, for example: “0.1.29d”. 
```

Comment:
change last sentence to `The version in the develop branch should include a suffix, for example: "0.1.29d" for differentiation.`